readme.md

```diff
@@ -1388,6 +1389,10 @@ export default () => <div>
 </div>
 ```
 
+### Starting a dev server on alternative IP
```

Comment:
It actually works for both `next` and `next start`.

Reply:
Labeling it as `Starting a server on alternative IP` would be better here ?

Reply:
`Starting the server on alternative hostname` and it'd be 👍

Reply:
Done. 👍